k8t/filters.py

```diff
@@ -68,8 +68,8 @@ def b64decode(value: Any) -> str:
 def hashf(value, method="sha256"):
     try:
         hash_method = getattr(hashlib, method)()
-    except AttributeError:
-        raise RuntimeError("No such hash method: {}".format(method))
+    except AttributeError as no_hash_method:
+        raise RuntimeError("No such hash method: {}".format(method)) from no_hash_method
```

Comment:
sexy

Reply:
pylint forced me hahaha

Reply:
looks pretty nice when you see the exception, output shows that the exception is directly related to AttributeError instead of only saying "this other exception happened as well"